platforms/core-runtime/launcher/src/main/resources/release-features.txt

```diff
@@ -0,0 +1,5 @@
+ - Configuration Cache is the preferred execution mode
+ - Java 17 is the new minimum required JDK version
+ - Kotlin 2.1 and Groovy 4.0 updates
+ - Semantic Versioning for Gradle releases
+ - Faster Kotlin DSL script compilation
```

Comment:
Kotlin script compilation isn't faster per se, but we're able to avoid compiling in more scenarios.
WDYT @eskatos 

```suggestion
 - Better Kotlin DSL build script compilation avoidance
```
or
```suggestion
 - Faster Kotlin DSL build script compilation when changing build logic
```
or
```suggestion
 - Shorter configuration time when making build configuration changes
```